themes/set-gnome-extensions-theme.sh

```diff
@@ -0,0 +1 @@
+gsettings set org.gnome.shell.extensions.tophat meter-fg-color $EXTENSIONS_TOPHAT_COLOR
```

Comment:
I would make this specific. I see the point that what if in the future we might have other extensions that could need styling, but that's future programming. Today we just have tophat. So should just frame all of this around tophat styling 👍

Reply:
So that would mean themes/*/tophat.sh and since it's only a single gsettings call, I would just do it inline in each of those files. Don't need the set-tophat-theme.sh helper.

Reply:
@dhh Done! 👍